CHANGELOG.md

```diff
@@ -7,25 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Updated inital offline package sync WSUS.cab.
-- Changed azurepipeline, use latest version of ubuntu and change the management 
+- Updated initial offline package sync WSUS.cab.
+- Changed azure pipeline to use latest version of ubuntu and change the management
   of pipeline artifact
 - Updated build.ps1 script and build.yaml.
 - Changed default timeout in Wait-Win32ProcessStart function for cab installation.
 - Updated pester test to support pester v5
 - Updated ReadMe.md to removed `RunRuleNow` parameter.
+- UpdateServicesApprovalRule
+  - Updated to load localization strings correctly.
+- UpdateServicesCleanup
+  - Updated to load localization strings correctly.
+- UpdateServicesServer
+  - Updated to load localization strings correctly.
 
 ### Added
 
 - Added TestKitchen files for integration tests
-- Added requiredmodules Sampler.GitHubTasks, powershell-yaml
+- Added required modules, Sampler.GitHubTasks, powershell-yaml
 - Added wildcard support in Products parameter of UpdatesServicesServer resource.
  (issue #13)
 
 ### Fixed
 
 - Fix deploy job in AzurePipeline, Added Sampler.GithubTasks in build.yaml
-- Fix issue #61 and #67, with add a foreach loop when `Set-TargetRessource` found 
+- Fix issue #61 and #67, with add a foreach loop when `Set-TargetResource` found
 multiple products dor the same `Title`.
```

Comment:
Did spot one extra error here ('dor' -> 'for') but didn't know if flagging it here would hold up the merge. I can fix in a later PR if necessary.

Reply:
Done